src/west/manifest.py

```diff
@@ -281,6 +282,10 @@ def _load(self, data):
             except ValueError as ve:
                 self._malformed(ve.args[0])
 
+            # Project names must be unique.
+            if project.name in project_names:
```

Comment:
Doesn't this fix a GitHub issue? I seem to recall there was one

Reply:
This is essential to the design of the multimanifest feature. Projects are identified by their names in that work, it's how we decide what projects to apply overrides of attributes like `url`, `name`, etc. to. This means it doesn't make sense to continue allowing two projects with the same "name" in a single manifest. Arguably it never did, but it was necessary before because some projects might have had the same final path component in a URL but different URL bases. That case can be handled now with an explicit `url:` for at least one of the two.

Reply:
Understood, thanks for the explanation